src/App.js

```diff
@@ -24,6 +25,28 @@ class App extends Component {
       .catch((error) => this.setState({ error }));
   };
 
+  handleChange = (event) => {
+    event.preventDefault()
+    this.setState({search: event.target.value});
+      let currentMovieList = [];
+      let newMovieList = [];
+  
+      if(event.target.value !== "") {
+        currentMovieList = this.state.homePageMovies;
+        newMovieList = currentMovieList.filter(movie => {
+          const movieTitle = movie.title.toLowerCase();
+          const userTitle = event.target.value.toLowerCase();
+  
+          return movieTitle.includes(userTitle);
```

Comment:
I like that the two titles are defined outside the return, makes it easier to read and see what the find is looking for.